game/src/ecs/components/AnimationComponent.java

```diff
@@ -10,8 +10,9 @@
 import semanticAnalysis.types.DSLTypeMember;
 
 /**
- * AnimationComponent is a component that stores the possible animations and the current animation
- * of an entity
+ * The AnimationComponent associates an entity with its animations. It stores the current animation
+ * and two idle animations, one for each direction (left/right). The current animation can be
```

Comment:
Das wirft grad die Frage auf, warum es kein "oben" und "unten" gibt? In die Richtungen kann man ja auch gehen? (Ist vermutlich aber kein Thema für diesen PR, sondern ein neues Issue.)

Reply:
Das hatte ich mir auch gedacht als ich das geschrieben habe.

Reply:
machst du ein ticket dafür auf und verlinkst es hier?

Reply:
Jup. Issue #606